docs/developers/ci.md

```diff
@@ -6,7 +6,8 @@ To generate this Matrix run `./tests/scripts/md-table/main.py`
 
 | OS / CNI | calico | cilium | custom_cni | flannel | kube-ovn | kube-router | macvlan |
 |---| --- | --- | --- | --- | --- | --- | --- |
-almalinux8 |  :white_check_mark: | :x: | :x: | :x: | :white_check_mark: | :x: | :x: |
+almalinux8 |  :white_check_mark: | :x: | :x: | :x: | :x: | :x: | :x: |
```

Comment:
That line should be removed by pre-commit if we don't have any test with almalinux 8, right ? Or did you leave one on purpose maybe ?

Reply:
I leave this one on purpose :-)